### DISCUSSION TITLE
Partial Match for worksheets

### DISCUSSION THREAD

**Discussion by amdcaseywilson** [dfinke/ImportExcel#1321] [Q&A]
Whats the syntax for a partial match worksheet name?
The worksheet has the date in the name like "11062022 Details"
Its the 2nd worksheet and columns conflict with the first worksheet.

This didnt treat the asterisk as a wildcard:
-WorkSheetName *Details

This had conflicting column headers:
-WorkSheetName *

Thank you,
Casey

**Comment by dfinke** [comment]
Please could you post a small repro
